src/altinn-app-frontend/src/utils/validation.test.ts

```diff
@@ -1,30 +1,34 @@
-import { createElement } from 'react';
-import { getInitialStateMock } from '../../__mocks__/initialStateMock';
-import { getMockValidationState } from '../../__mocks__/validationStateMock';
+import {createElement} from 'react';
+import {getInitialStateMock} from '../../__mocks__/initialStateMock';
+import {getMockValidationState} from '../../__mocks__/validationStateMock';
 import * as oneOfOnRootSchema from '../../__mocks__/json-schema/one-of-on-root.json';
 import * as refOnRootSchema from '../../__mocks__/json-schema/ref-on-root.json';
 import * as complexSchema from '../../__mocks__/json-schema/complex.json';
 
 import type {
-  IValidationIssue,
-  IValidations,
-  IRepeatingGroups,
-  IRuntimeState,
   IComponentBindingValidation,
   IComponentValidations,
   ILayoutValidations,
+  IRepeatingGroups,
+  IRuntimeState,
+  IValidationIssue,
+  IValidations,
 } from 'src/types';
-import type { ILayoutComponent, ILayoutGroup } from 'src/features/form/layout';
-
-import { Severity } from 'src/types';
-import { getParsedLanguageFromKey } from '../../../shared/src';
-import { createRepeatingGroupComponents } from 'src/utils/formLayout';
-import { getParsedTextResourceByKey } from 'src/utils/textResource';
+import {Severity} from 'src/types';
+import type {ILayoutComponent, ILayoutGroup} from 'src/features/form/layout';
+import {getParsedLanguageFromKey} from '../../../shared/src';
+import {createRepeatingGroupComponents} from 'src/utils/formLayout';
+import {getParsedTextResourceByKey} from 'src/utils/textResource';
 import * as validation from './validation';
-import { mapToComponentValidations } from './validation';
+import {mapToComponentValidations} from './validation';
 
 describe('utils > validation', () => {
   let mockLayout: any;
+  let mockGroup1: any; // Repeating group
```

Comment:
Yes and no.. I think group1 is a good enough name, and since the purpose of extracting it as a variable was to build simpler form layouts to test against, giving it a longer (but more descriptive) name somewhat defeats that purpose.

Have a look here:
https://github.com/Altinn/app-frontend-react/blob/0f0de91c175d2643d19f029dfdabad132e7a9938/src/altinn-app-frontend/src/utils/validation.test.ts#L832-L842

`mockGroup1` is not the best example, but I were to change the name of `mockComponent4` to `mockComponentInputInsideFirstRepeatingGroup` (or something to that effect), (in my opinion) it wouldn't add much to the code, but it would cause unwieldly long lines. I also don't see much point in renaming all the established component IDs in this mock layout, so if I named the variables more descriptively, the test code would make less sense (since I'm checking for component IDs later).

Also, I think we could do with a few more comments, not less.. :wink: